morphic.js

```diff
@@ -7519,6 +7519,10 @@ MenuMorph.prototype.addLine = function (width) {
     this.items.push([0, width || 1]);
 };
 
+MenuMorph.prototype.addSubmenu = function (key, submenu) {
+	this.addItem(key, key, null, null, null, null, null, submenu);
```

Comment:
ES6 would be cool, but we could do all that I wanted with just type checking arguments in plain es5. :)

Oooh I just called es5 plain. The web is so much nicer than it used to be!